presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java

```diff
@@ -272,6 +304,10 @@ public synchronized void start()
         for (int i = 0; i < runnerThreads; i++) {
             addRunnerThread();
         }
+        if (interruptRunawaySplits) {
+            long interval = Math.min(60, interruptSplitInterval);
```

Comment:
Why to cap at 60? Actually since it is not configurable why not simply hardcode 60 in the line below?

Reply:
This isn't meant to be configured by users it's just for the unit test so I can set it to 1 second so it completes quickly.